ivc/src/prover.rs

```diff
@@ -329,8 +329,8 @@ where
         }
     };
 
-    // we assume our folding degree is always 2, so number of chunks should be 1
-    let num_chunks: usize = 1;
+    // we interpolate over d4, so number of chunks should be 3
```

Comment:
what is `num_chunk` exactly? if this is about splitting the final constraint polynomials into sections that fit in d1, shouldn't it be 2/4? rather than 1/3.
Or is 1 chunk assumed to be zero?